### DISCUSSION TITLE
Discussion for handle value:block in feeds with remote:items

### DISCUSSION THREAD

**Discussion by thebells1111** [Podcastindex-org/podcast-namespace#485] [Clarification Needed]
With issues #363 #346 proposed by @agates the ability to create curated content lists as RSS feeds is attainable. The `<podcast:remoteItem>` allows a channel with a link to items found in other channels. However, the channel also allows a `<podcast:value>`  block. Now, most apps will override the channel level value block with the item level value block, but I can see a few issues arising, and would like some input to determine what these issues are, if these issues are worth addressing, and what we can do to fix them.

Here's some I've thought of:
1. An individual could make a list of episodes that don't have a `value` block, then is receiving the full value of anyone streaming to an episode they did very little produce. The listener would most likely assume all streaming and boosts are going to the podcaster, which wouldn't be the case.
2. An individual could be very skilled at curating content, and want to be included in the `value` block for a small percentage to cover the time and expertise it takes to curate a popular list. I don't necessarily have an issue with someone wanting to be included in the `value` block, but it brings up a whole host of other issues, like what's a proper percentage when curating, they'll be cutting into a podcaster's cut and the podcaster may not want to share revenue with someone for the simple task of curating a list, and if it's a popular podcast, a 1% cut could be a significant amount of money. Each person we add to the `value` block is at the very least a 1% cut of revenue to the podcaster.

I'm wondering if it's appropriate for a channel with `remoteItems` to have a channel level `value` block. I can see pros and cons to this.

I'm also thinking at some point value cuts will need the ability to have to be a set using something other than percentages. Something like a static fee of 10 sats is the curators cut of each transaction, or something like `10 sats or 1%, whichever is less`

**Comment by daveajones** [comment]
This is an interesting idea. I’ve been thinking that to move forward we would probably want to formalize remoteItem first since a lot of other things are starting to flow from that. Then we can include some other dependent features like we are discussing with lists, etc.

**Comment by daveajones** [comment]
@agates Do you have thoughts here?  I know we briefly discussed it in the board meeting, but I don't remember what the consensus was, or if there was one.

**Reply by agates** [reply]
My gut instinct is to allow a channel-level value block for the curator[s] in the playlist feed and the application can choose what % fee to allow to go to the curators.  Anything else seems like it would be open to abuse.
